dcm4che-core/src/test/java/org/dcm4che3/data/IDWithIssuerUnitTest.java

```diff
@@ -0,0 +1,84 @@
+package org.dcm4che3.data;
+
+import org.junit.Test;
+
+import java.util.Set;
+
+import static org.dcm4che3.data.Tag.*;
+import static org.junit.Assert.assertEquals;
+
+public class IDWithIssuerUnitTest {
```

Comment:
To be consistent with the other names in this package, the unit test class should be called "IDWithIssuerTest".

Please also add a class Javadoc with author (and maybe short explanaition and link to the class under test).